src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,22 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+
```

Comment:
please remove empty lines
same in other cases

Reply:
Not fixed